src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -149,6 +143,10 @@ export default function Swap({
     currencies,
     inputError: swapInputError,
   } = useDerivedSwapInfo()
+  const currencyIn = currencies[Field.INPUT]
+  const currencyOut = currencies[Field.OUTPUT]
+
+  const swapIsUnsupported = useIsSwapUnsupported(currencyIn, currencyOut)
```

Comment:
Since you are at it, can you rename it to `isSwapUnsupported`?

Reply:
Done: https://github.com/cowprotocol/cowswap/pull/1049/commits/2bedd82ee55c0413db58528a504715aa31fadb8b